### ISSUE TITLE
iANDI (not showing iframes always)

### ISSUE THREAD

**Issue by KSmithO** [SSAgov/ANDI#21]
Disagreement with a website and how ANDI iframes vs Frames favelet pick up iframes. The Twitter Timeline iFrame is in the tab order, you can use the focusable elements module in ANDI to get Accessible Name and Description for the iframe, but for some reason it isn’t showing up in the iframes module even though it is quite clearly an <iframe> element. Possibly a bug?
Website: http://cap.mil/ 

[ANDI iframe.docx](https://github.com/SSAgov/ANDI/files/2116232/ANDI.iframe.docx)

**Comment by SSAgov** [comment]
Thank you for finding the issue and providing a test page. An update to iANDI 1.0.3 addresses this issue. Users can now inspect iframes without src, but because of a technical browser limitation, when clicking on the "test in new window" button, a popup occurs which states "This iframe has no [src] and cannot be opened independently in a new window. ANDI cannot be used to test the contents of this iframe."
Iframes that are not visible, are intentionally ignored by ANDI since they do not require accessibility testing while they are not visible.
